app/src/main/java/org/simple/clinic/home/report/ReportsScreen.kt

```diff
@@ -64,6 +67,13 @@ class ReportsScreen(context: Context, attrs: AttributeSet) : FrameLayout(context
 
   override fun showReport(html: String) {
     showWebview(true)
+    webView.webViewClient = object : WebViewClient() {
+      override fun shouldOverrideUrlLoading(view: WebView?, request: WebResourceRequest?): Boolean {
+        val url = request?.url.toString()
```

Comment:
Since url can be nullable, calling `toString` will return a literal `null` value. Let's handle the `null` here.